src/tests/ftest/io/macsio_test.py

```diff
@@ -62,3 +68,41 @@ def test_macsio(self):
                 self.container.uuid))
         if status:
             self.log.info("Test passed")
+
+    def test_macsio_daos_vol(self):
+        """JIRA ID: DAOS-4983.
+
+        Test Description:
+            Purpose of this test is to check basic functionality for DAOS,
+            MPICH, HDF5, and MACSio with DAOS VOL connector.
+
+        Use case:
+            Six clients and two servers.
+
+        :avocado: tags=all,pr,hw,large,io,macsio_daos_vol
```

Comment:
Given that this is a "functionality" test, can it be run on VMs?

Reply:
Not sure ... @mchaarawi could/should MACSio run on VMs?

Reply:
macsio is definitely a benchmark and not a functional test.
it is compute and io intensive, so i would not make it a candidate to run on VMs.